### ISSUE TITLE
'vala-panel -c menu' should toggle menu, not just open it

### ISSUE THREAD

**Issue by AGitBoy** [rilian-la-te/vala-panel#79]
The command `vala-panel -c menu` only opens the menu, as opposed to toggling it. Changing this will enhance the ease of use to those that set the menu to open with a keybind.

**Comment by rilian-la-te** [comment]
Menumodel now just popups after timeout, not toggles a menu. Another applet can set different behaviour for this.

**Comment by rilian-la-te** [comment]
but is focuses a menu, it can be closed via Esc key.

**Comment by AGitBoy** [comment]
@rilian-la-te Maybe a way to close the menu from the command line?

**Comment by rilian-la-te** [comment]
But I do not understand why you need it. Can you explain use cases?

**Comment by AGitBoy** [comment]
@rilian-la-te Most menus are bindable to a key, in this case you can use the command line interface of Vala Panel to bind it to a key, but most menus when you hit the bound key again, it will close. Currently this is not possible in Vala Panel, as there is no way to either close the menu from the command line, or toggle the menu being open or closed from the command line.

**Comment by rilian-la-te** [comment]
But how you can interact with another parts of screen with opened menus? It is X after all)

**Comment by AGitBoy** [comment]
Like you said, until it is closed, it has focus. Thus, when it loses focus it closes.

**Comment by rilian-la-te** [comment]
Yes, then why you need a commandline menu deactivation, if you just can esc it?

**Comment by AGitBoy** [comment]
As I said before, the goal I am going for is toggling it with the meta key, if it's open and I git the meta key, it will close, if it's closed and I hit the meta key it will open.

**Comment by rilian-la-te** [comment]
It works on other panels such way?

**Comment by AGitBoy** [comment]
Most panels with popup menus integrated have this.
